doc/exits-and-hotspots.md

```diff
@@ -0,0 +1,58 @@
+# Hotspots and exits
+
+Hotspots in Escoria are thought of as "things a character can walk on", either
+like opening a door in Prince of Persia by stepping on a plate or a more abstract
+"To street" exit hotspot.
+
+Inert items, things that are "hotspots for the mouse" but not to be walked on, should
+be implemented using `item.gd`.
+
+Although hotspots and exits can be attached to any type of `Node`,
+not every case works properly.
+
+The working cases are described here.
+
+Exit is extended from hotspot. To make gameplay quicker and "more modern", hotspot
+implements double-clicking, which therefore works in exit as well.
+
+## Concerning the player
+
+Your player character must be a `KinematicBody2D` in order to work with hotspots
+and exits.
```

Comment:
`TextureRect` exits should work with `Node2D` player characters though.

Reply:
But maybe it's best not to mention it, since the `KinematicBody2D` player is easy to make and has this clear advantage.